src/actions.js

```diff
@@ -338,6 +338,31 @@ export function clearLastClaimAt() {
   };
 }
 
+export function fetchLastClaimWithSameDiagnosis(icd, chfid) {
+  const claimFilters = [
+    `chfid: "${chfid}"`,
+    `icd: "${icd.code}"`,
+    'status_Gt: 2',
```

Comment:
what does mean status Gt 2? Can we better describe in code by creating maybe const?

Reply:
If I'm not mistaken, this means that I want to receive claims with at least 'entered' status. I will add const to provide more precise naming.